### ISSUE TITLE
Could you show your training log？

### ISSUE THREAD

**Issue by encichou** [alfredgu001324/MapUncertaintyPrediction#7]
During training step， my Laplace loss from pos to neg。Could you show your training log？

**Comment by alfredgu001324** [comment]
Yes indeed that is the case because the loss function is now NLL. As the predicted distribution starts to match the GT in training, the NLL will keep decreasing to very negative values. 

[20231109_023039.log](https://github.com/user-attachments/files/15993339/20231109_023039.log)

**Comment by encichou** [comment]
3Q
